app/commands/task_creation.rb

```diff
@@ -0,0 +1,19 @@
+class TaskCreation
+  include ActiveModel::Model
+
+  attr_accessor :title, :description
+
+  validates :title,
+    presence: true,
+    length: { maximum: 40 }
+
+  validates :description,
+    length: { maximum: 255 }
```

Comment:
Style/AlignParameters: Align the parameters of a method call if they span more than one line.